hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestBlockTokens.java

```diff
@@ -377,6 +386,23 @@ private static void setSecureConfig() throws IOException {
         ozoneKeytab.getAbsolutePath());
   }
 
+  @Test
+  public void testRotateKeySCMAdminCommand()
```

Comment:
I think we need to move this test to a new test class. `TestBlockToken` is intended for block token test-cases and configure a relatively short secret key lifetime. This CLI test should be done with a longer rotation duration, e.g. 1h, so that the CLI and the automatic rotation don't collide.